millicore.go

```diff
@@ -0,0 +1,28 @@
+package main
+
+import (
+	"os/exec"
+	"path/filepath"
+)
+
+func GetMillicoreConfigTasks(tasks chan<- Task, runner Runner, projects []string, resourceFactory ResourceMatchFactory) {
+	for _, p := range projects {
+		pods, err := resourceFactory(p, "pod", "millicore")
+		if err != nil {
+			tasks <- NewError(err)
```

Comment:
yeah OK, that makes sense thinking about it, as Nagios will be showing quite a lot of errors if millicore is missing where it is expected.